enhancements/api-review/custom-tags-aws.md

```diff
@@ -48,77 +53,178 @@ Motivations include but are not limited to:
 - the administrator or service (in the case of Managed OpenShift) installing OpenShift can pass an arbitrary
    list of user-defined tags to the OpenShift Installer, and everything created by the installer and all other
    bootstrapped components will apply those tags to all resources created in AWS, for the life of the cluster, and where supported by AWS.
-- tags must be applied at creation time, in an atomic operation. It isn't acceptable to create an object and,
-   after some period of time, apply the tags post-creation.
+- tags can be applied at creation time, in an atomic operation.
+- tags can be updated post creation.
 
 ### Non-Goals
 
-- to reduce initial scope, tags are applied only at creation time and not reconciled. If an administrator manually
-   changes the tags stored in the infrastructure resource, behavior is undefined. See below.
 - to reduce initial scope, we are not implementing this for clouds other than AWS. We will not take any actions
    to prohibit that later.
 
+### Limitations
+
+- tags for PV volumes cannot be updated but only set during creation  of volume using `--extra-tags`.Present CSI spec does not support `UpdateVolume` to update volume metadata.
+   Restarting csi driver to initiate multiple `CreateVolume` will depend on underlying storage layer idempotency implementation for the operation. It is recommended to use a interface from CSI spec which enforces idempotency gurantees for volume metadata updates.
+   In case of AWS EC2  `CreateVolume`, returns `IdempotentParameterMismatch` error when identical request is not found for the same client token.
+   Also, the CSI spec `https://github.com/container-storage-interface/spec/blob/master/spec.md` mentions grpc error code
+   `6 ALREADY_EXISTS` for `a volume corresponding to the specified volume name already exists but is incompatible with the specified capacity_range, volume_capabilities, parameters, accessibility_requirements or volume_content_source`.
+
 ## Proposal
 
-New `experimentalPropagateUserTags` field added to `.platform.aws` of install config to indicate that the user tags should be applied to AWS
+New `propagateUserTags` field added to `.platform.aws` of install config to indicate that the user tags should be applied to AWS
 resources created by in-cluster operators.
 
-If `experimentalPropagateUserTags` is true, install validation will fail if there is any tag that starts with `kubernetes.io` or `openshift.io`.
+`propagateUserTags` field is by default set to true.
+`experimentalPropagateUserTags` field will be set for deprecation.
+
+If `propagateUserTags` is not set to false, install validation will fail if there is any tag that starts with `kubernetes.io` or `openshift.io`.
```

Comment:
I do not understand the constraint around this validation. The user cannot use a tag that starts with `kubernetes.io` or `openshift.io` **regardless** of whether the tags are propagated to resources created in the cluster.

Reply:
The constraint is as per the present implementation, following PR refers to the changes
https://github.com/staebler/installer/blob/2033421c8dcba057a643cdd35e040d7ddcd4c355/pkg/types/aws/validation/platform.go#L55

Is there another set of validation that disallows `kubernetes.io` or `openshift.io` by default?